README.adoc

```diff
@@ -114,15 +114,15 @@ Both sites will be maintained to promote the development and use of MicroProfile
 The https://wiki.eclipse.org/MicroProfile[MicroProfile wiki] contains up-to-date information about the project development, tools and other documentation.
 
 === Google Group
-The link:https://groups.google.com/forum/#!forum/microprofile[Microprofile Google Group] is the preferred place for discussions to take place.
+The link:https://groups.google.com/forum/#!forum/microprofile[MicroProfile Google Group] is the preferred place for discussions to take place.
 
 === Mailing Lists
-Besides the Google Group threads (main source of communication), there is also an link:https://dev.eclipse.org/mailman/listinfo/microprofile-dev[Eclipse microprofile-dev mailing list] which should be used for conversation specific to the Eclipse way of doing things.
+Besides the Google Group threads (main source of communication), there is also an link:https://dev.eclipse.org/mailman/listinfo/microprofile-dev[microprofile-dev mailing list] which should be used for conversation specific to the Eclipse way of doing things.
 By using this mailing list, the Eclipse team can more quickly recognize and respond to our requests. +
 
 Since the link:https://projects.eclipse.org/projects/technology.microprofile[MicroProfile project] is still in incubation, another useful mailing list is the link:https://dev.eclipse.org/mailman/listinfo/incubation[incubation mailing list] for asking general Eclipse process questions.
 
```

Comment:
Should we talk about the MicroProfile working group and its associated mailing list?  Not sure it's necessary since the individual Jakarta EE projects do not talk about the Jakarta EE Working Groups and its mailing lists.  But, since MP is a different type of community, I'm just asking the question.

Reply:
Good point, but I'd like to target next RC or before final to address. I'd like to finalize RC1 so we can push it out and get other release members up to speed on the mechanics of delivery a release.